Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Textarea.md

```diff
@@ -0,0 +1,27 @@
+# Textarea
+
+`Alias: Umbraco.TextboxMultiple`
+
+`Returns: String`
+
+Textarea is a simple HTML textarea control for multiple lines of text.
+
+## Data Type Definition Example
+
+![Textarea Data Type Definition](images/textarea/7/textarea-setup.png)
+
+
+## Settings
+
+## Content Example:
+
+![Textarea Content Example](images/textarea/7/textarea-content.png)
+
+
+## MVC View Example:
+
+	@{
+	   if (Model.Content.HasValue("textarea")){
+	       <p>@(Model.Content.Textarea</p>
```

Comment:
This bit is missing the closing round bracket...

... also got me thinking whether the example should use an alias that is a different name to the DataType, in case that led to some confusion?

Also in the example, if we're using HasValue, should the example use GetPropertyValue... or is there a Modelsbuilder equivalent to HasValue?

Reply:
Good point Mark - I'll change the alias and fix the missing bracket.

I'll also modify the example to using GetPropertyValue like it does in Textbox https://our.umbraco.com/documentation/Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Textbox

Reply:
Just made a new commit with the suggested fixes